build.sbt

```diff
@@ -33,7 +33,7 @@ val observabilityDependencies = Seq(
   "com.softwaremill.sttp.tapir" %% "tapir-opentelemetry-tracing" % tapirVersion, // OTEL <-> Tapir integation
   "com.softwaremill.ox" %% "otel-context" % oxVersion, // OTEL context propagation in Ox scopes
   "io.opentelemetry" % "opentelemetry-exporter-otlp" % otelVersion exclude ("io.opentelemetry", "opentelemetry-exporter-sender-okhttp"),
-  "io.opentelemetry" % "opentelemetry-exporter-sender-jdk" % otelVersion,
+  "io.opentelemetry" % "opentelemetry-exporter-sender-okhttp" % otelVersion,
```

Comment:
we shouldn't need this chnage - I wanted to avoid that dep explicitly in the step above

Reply:
Reverted